BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/BetterShardsListener.java

```diff
@@ -489,4 +493,31 @@ public void blockPlaceEvent(BlockPlaceEvent event) {
 			event.setCancelled(true);
 		}
 	}
+	
+	@EventHandler(priority = EventPriority.LOWEST)
+	public void portalLoadEvent(PortalLoadEvent event) {
```

Comment:
Listening to your own events, especially when that event is only created in one place, is bad practice